### PR TITLE
Adds a toy weapon vendor into the syndie lavaland base firing range

### PR DESCRIPTION
**Pull request by ma44**
:cl: ma44
add: Reports of syndicate base on lavaland has been outfitted with a state of the art donksoft toy weapon dispenser. 
/:cl:

https://i.gyazo.com/9894cb6ecfd67c86c80e4519ca494844.png

[why]: Adds some more weapons to the shooting range, as there's usually only a c20r and pistols, and this adds some other weapons and toys to it with a vendor. 

If you have any suggestions for what the vendor should say, that be great.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Donksoft Vendor.

When it's hacked it should give out boxes of riot darts or something.

**Comment by ma44** [comment]
Mostly more about what the vendor says as an advertisement, something like"Trigger a valid hunter today!" doesn't sound that great and memey

**Comment by Armhulen** [comment]
MAKE A SYNDICATE VENDING SPRITE YOU BUM.

**Comment by ma44** [comment]
re

**Comment by Armhulen** [comment]
let me see if i can do something

no garentees

**Comment by Armhulen** [comment]
replace the katana, that's not syndicate. Maybe make it the toy esword for memes

**Comment by ma44** [comment]
We already have a toy, i slapped in a katana as it actually does 5 damage and thus makes it 18+ (reference to riot darts description)

**Comment by Armhulen** [comment]
great, I made sprites but i dunno if they're up to muh44 standards
http://i.imgur.com/J9gc47N.png

**Comment by Armhulen** [comment]
don't worry, it doesn't just hover over kill. It swaps between weapons with messages flashing inbetween
this compares it to all the other vendors
http://i.imgur.com/twm2j6n.png

**Comment by ma44** [comment]
Wew that's pretty good

Maybe i could make a subtype that's admin spawn only, but it spawns ACTUAL weapons and equipment

**Comment by Armhulen** [comment]
ok great 
https://www.dropbox.com/s/4zjm9iv9fj1wofk/syndi.dmi?dl=0
here it is if you wanna YUSE it

**Comment by juanototo2** [comment]
needs a double bladed foam sword in contraband mate.

**Comment by ma44** [comment]
Ill just go throw in some double sabers into the contraband (even tho you can already make 5 with regular swords) contraband setting is now the nook ops setting :honk:

**Comment by Iamgoofball** [comment]
>Maybe i could make a subtype that's admin spawn only, but it spawns ACTUAL weapons and equipment

this is literally already a thing

**Comment by ma44** [comment]
Obviously it needs to dispense marauders

**Comment by Armhulen** [comment]
The adminsprite is a placeholder you donk

**Comment by Armhulen** [comment]
YES SOMETHING OF MINE IS IN THE GAME FUCK YOU ALL I WIN

**Comment by Incomptinence** [comment]
Just noting this accidentally removed the viruses from the smart fridge that were put back in there in: 
https://github.com/tgstation/tgstation/pull/27289

File: _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm [hunk 1]
```diff
@@ -729,14 +725,22 @@
 	dir = 8
 	},
 /area/ruin/powered/syndicate_lava_base)
-"bN" = (
-/obj/machinery/smartfridge/chemistry/virology/preloaded,
+"bM" = (
+/obj/machinery/smartfridge/chemistry/virology,
```
Comment (Penguaro) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm]: You monster...

**Pull request merged**